source/packages/libraries/clients/device-patcher-client/CHANGELOG.md

```diff
@@ -1,81 +1,76 @@
 # Change Log - @awssolutions/cdf-device-patcher-client
 
-This log was last generated on Thu, 09 Mar 2023 22:35:00 GMT and should not be manually modified.
+This log was last generated on Tue, 15 Aug 2023 18:35:38 GMT and should not be manually modified.
 
-## 0.0.0
+## 2.4.0
```

Comment:
This one also has a strange sequence of versions

Reply:
Yes, this is because this was not previously external. Because of our change in development, we have promoted many of the packages from internal to external. This one is happening in this release so it's going from the internal version of 0.0.0 to the external version 2.4.0.